doc/format.rst

```diff
@@ -729,17 +740,27 @@ Other points to keep in mind
 
       `Example <http://www.example.com>`_
 
-
 Common reST concepts
 --------------------
 For paragraphs, indentation is significant and indicates indentation in the
 output. New paragraphs are marked with a blank line.
 
-Use ``*italics*``, ``**bold**`` and ````monospace```` if needed in any
-explanations
-(but not for variable names and doctest code or multi-line code).
-Variable, module, function, and class names should be written between
-single back-ticks (```numpy```).
+Use ``*italics*``, ``**bold**`` if needed in any explanations.
+
+Use of backticks in reST is a common point of confusion it is different from
+markdown:
+
+- Module, function, and class names should be enclosed within ```single
+  backticks```. These are intended to render as hyperlinks (e.g. `numpy`). If
+  the hyperlinks do not render as intended, [insert reference to documentation
+  here].
+- References to parameters, attributes, and methods defined within the same
+  docstring should be enclosed within ```single backticks```. These are
+  currently rendered in *italics*, but there are plans to render a
+  ``monospaced`` hyperlink to the relevant definition in a future version of
+  numpydoc.
```

Comment:
Perhaps this should be removed, but I wanted to mention it because it is the only reason I can see for using single backticks instead of double backticks. Otherwise, use of single backticks only causes problems with name collisions and renders in italics (which usually means "emphasis") where monospace (which means code) would be more clear.

Reply:
I'd just remove this - it's probably more detail than most users are interested in (and is not necessarily correct - the actual rendering of e.g. italics is actually dictated by the sphinx theme).